app/src/test/java/gov/va/vro/controller/VroControllerTest.java

```diff
@@ -56,8 +56,11 @@ class VroControllerTest extends BaseControllerTest {
 
   @Autowired protected CamelContext camelContext;
 
-  @EndpointInject("mock:claim-submit-full")
-  private MockEndpoint mockFullHealthEndpoint;
+  @EndpointInject("mock:rabbit-claim-submit")
```

Comment:
renamed to be *very* explicit what we are actually routing around. it's *NOT* the full camel route (that has the same name).

Took far too long to figure out what this file was doing with the other name.